src/create-or-update-comment.ts

```diff
@@ -125,6 +125,12 @@ async function createComment(
   issueNumber: number,
   body: string
 ): Promise<number> {
+  // 65536 characters is the maximum allowed for issue comments.
+  if (body.length > 65536) {
```

Comment:
This works great for comment creation, but unfortunately doesn't work for the update replace and update append modes

Reply:
I opened a PR: #205